.github/workflows/graphql.yml

```diff
@@ -0,0 +1,40 @@
+name: GraphQL API Documentation
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'internal/graphql/schema/schema.graphql'
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'internal/graphql/schema/schema.graphql'
+jobs:
+  test:
+    runs-on: ubuntu-latest

```

Comment:
@amino19 Will you also be able to add a condition here so that this job gets triggered only when any of these activities happen on `'meshery/meshery'` repo? :slightly_smiling_face:

Reply:
I've updated the suggestion here

Reply:
@amino19 This is where you'll be placing the condition on `meshery/meshery`

Reply:
@DelusionalOptimist : Got it, Thanks! :slightly_smiling_face:

Reply:
Great :+1: